en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-1100-to-400.md

```diff
@@ -4535,7 +4533,7 @@ Follow the instructions below to move all the existing API Manager configuration
         
         3. Migrate to the new throttling engine and generate throttle policies. 
         
```

Comment:
this step needs to verify and add if required. Because in 4.0.0 we do not keep the synapse configs in the file system. @tharindu1st thoughts?